src/main/resources/reference.conf

```diff
@@ -47,6 +47,7 @@ jdbc-journal {
 
   slick {
     profile = "slick.jdbc.PostgresProfile$"
+    profile = ${?jdbc-journal.slick.driver}
```

Comment:
I did some local test to simulate the case in which an user have configured `driver` and we kind of move it to `profile`. 

That will only work if `driver = slick.jdbc.PostgresProfile$`.

If the user has it as `driver = slick.driver.PostgresDriver$`, we are in trouble. Because Slick will see it as the value of `profile` property and try to load `slick.driver.PostgresDriver`. And that will fail because there is not such a class anymore, only a type alias.

Reply:
I see your point. In that case it would actually be better to not include this setting.

In the end it might actually make sense to remove all references to postgress at all. In this case a misconfigured database configuration would fail to load (hopefully with a useful error message). Removing this default would actually prevent issues such as #116

Reply:
It's true that the `reference.conf` should not contain any settings that are specific to environment. I think that it's often used as a reference for documentation, but that's not the best thing to do because we end up setting default values that can be very confusing.